packages/teams-js/src/internal/constants.ts

```diff
@@ -143,6 +143,8 @@ export const validOrigins = [
   'powerpoint.office.com',
   'www.officeppe.com',
   '*.www.office.com',
```

Comment:
not related to this PR but shall this line be put before line 140?

Reply:
Sure thing